src/index.js

```diff
@@ -0,0 +1,168 @@
+  return moviesAndlikes;
+};
+
+export default getShows;
+
+const updateShowList = async () => {
+  const shows = await getShows();
+
+  const showsList = shows[0];
+  const likeList = shows[1];
+  showItems.innerHTML = '';
+  showsList.forEach(async (show) => {
+    const showElement = document.createElement('div');
+    showElement.classList.add('show-item');
+
+    showElement.innerHTML = `
+    <img class="show-image" src="${show.image.medium}" alt="${show.name}" />
+    <div class="details">
+    <span class="show-title">${show.name}</span>
+    <button class="like-button" data-show-id=${show.id}>
+      <svg viewBox="0 0 24 24" fill="white">
+        <path d="M12,21.35L10.55,20.03C5.4,15.36 2,12.27 2,8.5C2,5.41 4.42,3 7.5,3C9.24,3 10.91,3.81 12,5.08C13.09,3.81 14.76,3 16.5,3C19.58,3 22,5.41 22,8.5C22,12.27 18.6,15.36 13.45,20.03L12,21.35Z" />
+      </svg>
+    </button>
+    ${
+      likeList.find((like) => like.item_id === show.id)
+        ? `<span class="like-count">${
+            likeList.find((like) => like.item_id === show.id).likes
+          }</span>`
+        : ''
+    }
+  </div>
+
+    <div>
+    <button  class="comment-button">Comment</button>
+    </div>
+    `;
+    const h3 = document.querySelector('.tv-title');
+    showItems.appendChild(showElement);
+    const count = document.querySelectorAll('.show-item');
+    itemCounter(h3, count);
+    const likeBtn = showElement.querySelector('.like-button');
+    likeBtn.addEventListener('click', async () => {
+      likeBtn.classList.toggle('liked');
+      await addLike(show.id);
+      updateShowList();
+    });
+
+    const commentBtn = showElement.querySelector('.comment-button');
+    commentBtn.addEventListener('click', async () => {
+      const comments = await getComments(show.id);
+
+      const tvshow = document.querySelector('.popup');
+      const title = document.querySelector('.tv-title');
+      title.style.display = 'none';
+      showItems.style.display = 'none';
+      tvshow.innerHTML = `
+       <div class="comment-popup">
+       <img class="show-image" src="${show.image.medium}" alt="${show.name}" />
+       <button class="close-icon">X</button>
+       </div>
+        <div class="details">
+        <span class="show-title">Title: ${show.name}</span>
+        <div class="show-description"><span>DESCRIPTION:</span> ${
+          show.summary
+        }</div>
+       <div class="list">
+       <ul class="show-genres">
+       <li class="rating">Rating: ${show.rating.average}</li>
+       <li class="duration">Duration: ${show.runtime} minutes</li>
+       <li class="season">Status: ${show.status}</li>
+       </ul>
+       </div>
+        <div class="container">
+        <div class="comments">
+        <h2>Comments <span class="comment-count">${comments.length}</span></h2>
+        <ul class="comment-list">
+        ${comments
+          .map(
+            (comment) =>
+              // eslint-disable-next-line implicit-arrow-linebreak
```

Comment:
- [ ] Please remove this linters disablement line and fix all eslint issues yourself.